sql-injection/activities/demo-sql-injection/README.md

```diff
@@ -0,0 +1,28 @@
+# Name: Demo
+
+## Description
+
+This is a simple demo that will be used as an example in the session.
+
+Similar, but not the same. Can you find the name of the bastard?
+Check this out: http://ctf-18.security.cs.pub.ro:8083/
+
+Score: 25
+
+## Vulnerability
+
+SQL injection vulnerability.
+
+## Exploit
+
+Search in the `flags` table for the flag.
+
+## Environment
+
+Web server with PHP and MYSQL support. (deployable as a Docker container using files in `deploy/` folder)
+
+## Deploy
+
+Copy `deploy/` folder and run `make run`.
```

Comment:
Should mention that first one must run the `make` command in `src` and then `make run` in `deploy/`. This clarification should be done for all activities, but I won't leave comments at all of them.